pipeline/stage02_processing/scripts/spatial_downsampling.py

```diff
@@ -42,48 +36,40 @@ def spatial_smoothing(asig, MACRO_PIXEL_DIM, DIM_X, DIM_Y):
 
     CLI.add_argument("--macro_pixel_dim",  nargs='?', type=int,
                       help="smoothing factor", default=2)
-    CLI.add_argument("--dim_X",  nargs='?', type=int,
-                      help="original x dimension", default=100)
-    CLI.add_argument("--dim_Y",  nargs='?', type=int,
-                      help="original y dimension", default=100)
-    
+
     
     args = CLI.parse_args()
 
     block = load_neo(args.data)
-    asig = block.segments[0].analogsignals[0]
+    block = AnalogSignal2ImageSequence(block)
+    imgseq = block.segments[0].imagesequences[-1]
```

Comment:
Imagesequences are for now not stored when the neo block is written to file. Therefore, the imagesequences list is empty. So, in case there are multiple analogsignals, here only the first imagesequence object should be selected. Thus, `imgseq = block.segments[0].imagesequences[0]`